src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -72,7 +76,7 @@ const ZUIOrganizeSidebar = (): JSX.Element => {
   const messages = useMessages(messageIds);
   const classes = useStyles();
   const theme = useTheme();
-
+  const user = useCurrentUser();
```

Comment:
Some component tests are failing because of the introduction of this hook. By using this hook, the code will require the store provider, and maybe that hasn't been added to the test renderer?